internal/host/host.go

```diff
@@ -205,6 +205,18 @@ func (m *Manager) RegisterHost(ctx context.Context, h *models.Host, db *gorm.DB)
 		if h != nil {
 			host.Kind = h.Kind
 		}
+
+		// If the host already exists in the DB, we update its registration timestamp to track
+		// the subsequent registration events. This can happen if the host rebooted or the agent has
+		// been restarted.
+		updates := make(map[string]interface{})
+		updates["registered_at"] = strfmt.DateTime(time.Now())
+		if err := db.Model(&common.Host{}).Where("id = ? and infra_env_id = ?", host.ID, host.InfraEnvID).Updates(updates).Error; err != nil {
```

Comment:
If we're only updating a single column is there any reason not to use `.Update` rather than `.Updates`?

ref: https://gorm.io/docs/update.html#Update-single-column

Reply:
Yeah, that makes more sense